src/tiktok_uploader/upload.py

```diff
@@ -157,20 +202,31 @@ def _go_to_upload(driver) -> None:
     ----------
     driver : selenium.webdriver
     """
-    logger.debug(green('Navigating to upload page'))
+    logger.debug(green("Navigating to upload page"))
 
-    driver.get(config['paths']['upload'])
+    driver.get(config["paths"]["upload"])
 
     # changes to the iframe
     iframe_selector = EC.presence_of_element_located(
-        (By.XPATH, config['selectors']['upload']['iframe'])
+        (By.XPATH, config["selectors"]["upload"]["iframe"])
+    )
+    iframe = WebDriverWait(driver, config["explicit_wait"]).until(iframe_selector)
+
+    creator_center_iframe_selector = EC.presence_of_element_located(
+        (By.XPATH, config["selectors"]["upload"]["creator_center_iframe"])
+    )
+
+    try:
+        creator_center_iframe = WebDriverWait(driver, config["explicit_wait"]).until(
+            creator_center_iframe_selector
```

Comment:
The meat of the Pull Request. Will implement this as a standalone change when I have the time. This definitely should be in code, I just got lazy when writing this and put an `explicit_wait`